setup.py

```diff
@@ -25,10 +25,9 @@
         'Intended Audience :: Science/Research',
         'Topic :: Scientific/Engineering :: Bio-Informatics',
         'Natural Language :: English',
-        'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
         'Programming Language :: Python :: 3.7',
         'Programming Language :: Python :: 3.8',
     ],
-    install_requires=['connexion', 'Flask-Cors', 'Flask-PyMongo'],
+    install_requires=[],
```

Comment:
The version string (further up, but couldn't add the comment there) should only be hard-coded in a single place, so please make use of `drs_filer/__init__.py` like we are now doing in FOCA.